scripts/upgradeToV2/upgradeToV2.ts

```diff
@@ -556,6 +556,7 @@ export const main = async (networkName = name) => {
   };
 
   const deployStakingContracts = async release => {
+    const isRopsten = networkName !== "fuse-mainnet";
```

Comment:
ropsten is fuse-mainnet and staging-mainnet

Reply:
added